src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -243,15 +243,15 @@ struct MapRule
 
     const std::string match(const std::string &sub,
                             const std::string &username,
-                            const std::string &req_path,
+                            const std::string_view &req_path,
```

Comment:
I'd prefer to keep taking a reference to string here. One argument is for consistency: why take only this one as `std::string_view` and keep the others as `std::string&`? Ideally, one should also choose between taking either a reference to string, or a `std::string_view` by value, taking a reference to a `std::string_view` doesn't make a lot of sense. Finally, although it's not the case here, `std::string_view` is often [misused](https://github.com/root-project/root/pull/937/commits/7169aa5bb2e1064829d52b35cc4bccfefac3b4aa) in the sense that people take in a `std::string_view` only to then create a `std::string` from it, which does exactly what we'd like to avoid when using `std::string_view` in the first place: copying over the data.